internal/stm32CubeMX/stm32CubeMX.go

```diff
@@ -757,22 +762,42 @@ func GetStartupFile(outPath string, bridgeParams BridgeParamType) (string, error
 	}
 
 	var startupFile string
+	var defaultStartupFile string
+	var startupFileList []string
+
 	err = filepath.Walk(startupFolder, func(path string, f fs.FileInfo, err error) error {
 		if f.Mode().IsRegular() &&
-			strings.HasSuffix(f.Name(), fileExtesion) &&
-			strings.HasPrefix(f.Name(), "startup_") {
-			if fileFilter != "" {
+			strings.HasPrefix(f.Name(), "startup_") &&
+			(strings.HasSuffix(f.Name(), fileExtensions[0]) ||
+				strings.HasSuffix(f.Name(), fileExtensions[1]) ||
+				strings.HasSuffix(f.Name(), fileExtensions[2])) {
+
```

Comment:
Shouldn't the fileExtensions array better be a map instead of const indexing? What if somebody extends the array but not thinking of this constant indexing?

Reply:
I completely agree. Please check proposed change: [Commit](https://github.com/Open-CMSIS-Pack/generator-bridge/pull/100/commits/a0ced1e339e757ecbda4d00392114c213c71a38c)

Thanks

Reply:
Very good, this looks much better and is safer to maintain.